scss/components/time.scss

```diff
@@ -11,17 +11,13 @@ $block: #{$fd-namespace}-time;
       max-width: fd-space(10);
       text-align: center;
       margin-right: fd-space(1);
+
       &:last-child {
         margin-right: 0;
       }
     }
 
     &__input {
-        margin: fd-space(2) 0;
-        input {
```

Comment:
Most primary change—the removal of this padding messes things up, but I'm not sure where to put it instead.

Reply:
🤔  You are right that these styles are specific to this context.  We have n-number of "items", each with an input and two buttons.  Let's style it that way.  Let's keep the `fd-time__item` styles.  Then, let's remove (🔥) the current `fd-time__input` wrapper `<div>` and put that class directly on the `<input>` and make the styling:
```css
&__input {
    margin: fd-space(2) 0;
    padding-left: fd-space(2);
    padding-right: fd-space(2);
    text-align: center;
}
```
I would also put the `fd-time__input` as a child selector to `fd-time__item` to make sure our specificity wins on the `<input>` (since that will have styles already coming from `fd-form__control`).

This will all be a breaking change, but hey -- we're on version 0. 😄

Reply:
This makes a lot of sense, good suggestion on just repurposing the class (maybe that's what Jenna was hinting at earlier and I didn't fully grasp it).